docs/src/main/paradox/java/http/routing-dsl/directives/index.md

```diff
@@ -83,4 +83,38 @@ transformations, both (or either) on the request and on the response side.
 
 ## Composing Directives
 
-TODO rewrite for Java API
```

Comment:
wow that's embarrassing that we had a TODO left like this...
Thanks a ton @jlprat !